app/agents/git_hub_agent.rb

```diff
@@ -134,6 +134,25 @@ def remove_user_from_team(team_name, username)
     )
   end
 
+  def get_status(name)
+    full_name = "#{@org}/#{name}"
+    status_branch = 'master'
+    client = app_installation_client
+
+    response = client.status(full_name, status_branch)
+
+    statuses = []
+    response[:statuses].each do |status|
```

Comment:
```suggestion
    Array(response[:statuses]).flatten.each do |status|
```

… in case the field `statuses` does not exist, or is `nil`.